+++
title = "Web Implementation of the HBV Dual Infection Model"
date = 2011-09-29
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Matthias DÃ¶ring"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["4"]

# Abstract and optional shortened version.
    abstract = """Hepatitis B is endemic in many parts of the world and each year approximately 600 000 people die as a consequence of infection with hepatitis B virus (HBV). HBV genomes are divided into eight genotypes A to H according to phylogenetic analysis. Genotyping of HBV is an important tool in tracking the evolution of the viral genome as well as in therapy. It is possible for one organism to be infected by HBV viruses of different strains. Such an infection is referred to as a HBV dual infection. The identification of dual infections could improve results in the areas mentioned above. Although several computational methods for genotyping beside phylogenies have been developed in the past, none of them was explicitly concerned with HBV dual infections.

In this thesis, we present the implementation of a web tool that is able to identify the genotype(s) of one or multiple HBV nucleotide sequences, either full or subgenomic. Its main feature is the ability to recognize HBV dual infections. In this approach, HBV genotypes are predicted according to a probabilistic model that is based on genotype-specific nucleotide distributions. A maximum likelihood classifier is used to determine the most likely genotype. Our method was able to identify the genotypes of all 2258 annotated single infection sequences in a data set retrieved from NCBI correctly. Recombinant genotypes were correctly identified in 46 (86.8%) of 53 annotated recombinant sequences. To determine the performance for dual infections, an artificial data set of 6482 sequences was generated by pairwise mixing of sequences. For this data set, dual infections of HBV with different genotypes could be correctly identified in 5206 (99.9%) of 5211 such sequences.

This tool enables the detection of HBV dual infections, single infections, and recombinants in a reliable and fast manner. It may be of use to basic research and may help to further therapy and tracking of viral evolution. It will be available on the geno2pheno[hbv] web server, which is located at http://hbv.bioinf.mpi-inf.mpg.de/index.php."""

# Is this a selected publication? (true/false)
selected = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
# projects = ["geno2pheno-ngs"]

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Links (optional).
url_pdf = "web_implementation_of_the_hbv_dual_infection_model_doering_2011.pdf"
url_project = "http://hbv.bioinf.mpi-inf.mpg.de/"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
#url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Digital Object Identifier (DOI)
doi = ""

# Does this page contain LaTeX math? (true/false)
math = true

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = "Image credit: [**Unsplash**](https://unsplash.com/photos/pLCdAaMFLTE)"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""
+++
